### PR TITLE
* #20: Added a user guide.

### LINKED ISSUE #20: Create a short user guide
## Problem

We are missing a user guide.
We would like to add short documentation about how to install an in-development connector for evaluation.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #20

### CODE REVIEW

File: doc/user_guide/user_guide.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+The Exasol Tableau Connector is distributed together with Tableau Desktop and Tableau Server applications. We recommend using the latest available version of Tableau products to access the connector.
+If you want to use the currently developed version of connector, you can follow the guide below and install the connector disabling sign verification.
+
+## Testing In-Development Connector
+
+## With Tableau Desktop
+
+* Clone the Tableau-Connector repository. You can execute the following command from [Git Bash](https://gitforwindows.org/) terminal:
+
+```
```
Comment (jakobbraun) [inline doc/user_guide/user_guide.md:12]:
````suggestion
```shell
````
Also for otherts

Comment (jakobbraun) [reply doc/user_guide/user_guide.md:12]: Indent the snippets so that markdown shows them in the enumeration

Comment (jakobbraun) [reply doc/user_guide/user_guide.md:12]: also for all other.

File: doc/user_guide/user_guide.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+* Clone the Tableau-Connector repository:
+
+```
+git clone https://github.com/exasol/tableau-connector.git
+```
+
+* Package the connector with the script we provide (requires python installed):
+
+```
+/bin/bash <path to connector directory>/tableau-connector/tableau-server-GUI-tests/set_up_scripts/package_connector.sh
```
Comment (jakobbraun) [inline doc/user_guide/user_guide.md:39]: that won't work on windows. I guess that most percent of the tableau users use windows, so we should maybe also add a guide for them.

Comment (jakobbraun) [reply doc/user_guide/user_guide.md:39]: Or at least mention that you need a linux for those steps

**Review by jakobbraun — CHANGES_REQUESTED** [review]

File: doc/user_guide/user_guide.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide/user_guide.md:39]: This is the command I used on Windows actually. I've mentioned that I use Git bash, maybe I can add a note here again about it

File: doc/user_guide/user_guide.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+## Testing In-Development Connector
+
+## With Tableau Desktop
+
+* Clone the Tableau-Connector repository. You can execute the following command from [Git Bash](https://gitforwindows.org/) terminal:
+
+```shell
+git clone https://github.com/exasol/tableau-connector.git
+```
+
```
Comment (jakobbraun) [inline doc/user_guide/user_guide.md:15]:
````suggestion
* Clone the Tableau-Connector repository. You can execute the following command from [Git Bash](https://gitforwindows.org/) terminal:

    ```shell
    git clone https://github.com/exasol/tableau-connector.git
    ```

````

**Review by jakobbraun — CHANGES_REQUESTED** [review]

**Review by jakobbraun — APPROVED** [review]
:+1:

**Pull request merged**